LiskHQ/lisk
# Remove binary block serialization - Closes #1621

### What was the problem?

Blocks were being serialized to BSON before being broadcast to peers - This didn't offer any benefits in terms of performance.

### How did I fix it?

Removed BSON serialization and deserialization and added test cases to `test/unit/modules/transport.js`.

### How to test it?

npm run mocha test/unit/modules/transport.js
npm run mocha test/unit/modules/blocks.js
npm run mocha test/unit/modules/blocks/api.js
npm run mocha test/unit/modules/blocks/chain.js
npm run mocha test/unit/modules/blocks/process.js
npm run mocha test/unit/modules/blocks/utils.js
npm run mocha test/unit/modules/blocks/verify.js

The PR closes #1621
